apps/dashboard/src/app/(dashboard)/layout.tsx

```diff
@@ -4,9 +4,9 @@ import { DashboardHeader } from "../components/Header/DashboardHeader";
 export default function DashboardLayout(props: { children: React.ReactNode }) {
   return (
     <>
-      <div className="flex flex-col h-full px-4">
+      <div className="flex flex-col h-full">
         <DashboardHeader />
-        <main className="grow">{props.children}</main>
+        <main className="px-4 grow">{props.children}</main>
```

Comment:
why this change?

Reply:
The header's bottom border didn't stretch across the entire screen

Reply:
ah makes sense, that said we gotta make sure the "body" width (including padding) and header with end up the same, probably just need to used standardized "container" definitions everywhere